pkgs/servers/imgproxy/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, buildGoModule, fetchFromGitHub, pkg-config, vips, gobject-introspection }:
+
+buildGoModule rec {
+  pname = "imgproxy";
+  version = "2.8.1";
+
+  src = fetchFromGitHub {
+    owner = "imgproxy";
+    repo = "imgproxy";
+    sha256 = "00hhgh6nrzg2blc6yl8rph5h5w7swlkbh0zgsj7xr0lkm10879pc";
+    rev = "v${version}";
+  };
+
+  buildInputs = [
+    gobject-introspection
+    pkg-config
+    vips
+  ];
+
+  preBuild = ''
+    export CGO_LDFLAGS_ALLOW='-(s|w)'
+  '';
+
+  goPackagePath = "github.com/imgproxy/imgproxy";
+
+  modSha256 = "0kgd8lwcdns3skvd4bj4z85mq6hkk79mb0zzwky0wqxni8f73s6w";
```

Comment:
Please put `modSha256` below the `src` definition.

Reply:
I've just dropped them and running test build on my machine. Should I bring them back?

Reply:
Sorry, my deleted comment was of course an error, I meant that `goPackagePath` can be removed.